src/transCore.tsx

```diff
@@ -30,6 +30,12 @@ export default function transCore({
 }): Translate {
   const { logger = missingKeyLogger } = config
 
+  // By default, keys translated to empty strings
+  // are translated to empty strings or fallbacks/default options.
+  // If allowEmptyStrings is passed as false, result is
+  // the key itself.
+  const allowEmptyStrings = config.allowEmptyStrings || true
```

Comment:
Could we change the default to be false so that it doesn't break what we have so far? Thanks!

Reply:
Hey @aralroca. 
Maybe my comment there is not totally clear but the current implementation allows empty strings. Therefore `true` by default follows what we have so far.
Please have a look on the tests where I've described all the scenarios.

Reply:
Hey @aralroca.

Maybe it's not clear by looking at the code but the current implementation allows empty strings. Therefore `true` by default doesn't break what we have so far. `false` would.

I've described all the scenarios in test cases. Please have a look.
Also on README.md I tried to make it clear:

allowEmptyStrings`   | Change how translated empty strings should be handled. If omitted or passed as true, it returns an empty string. If passed as false, returns the key name itself (including ns). | `Boolean` | `true`

Reply:
@vascosilvaa Absolutely right. I had confused now when it is not in the dictionary that now shows the key, but it is true that if it exists but it is empty then it does print the empty string. Ignore what I said then. That's enough! Thank you!